src/org/usfirst/frc4904/robot/commands/SetOverride.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class SetOverride extends Command {
+	public boolean override;
+	public boolean setValue;
+	public OverridableCommand command;
+
+	public SetOverride(boolean setValue, OverridableCommand command) {
```

Comment:
Maybe make this able to take in as many commands as you want? If not, flip the order of these arguments.